tests/Integration/Schema/Directives/ExtendsDirectiveTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Tests\Integration\Schema\Directives;
+
+use Illuminate\Testing\Fluent\AssertableJson;
+use Tests\TestCase;
+
+class ExtendsDirectiveTest extends TestCase
+{
+    public function testCanExtendTypes()
+    {
+        $this->mockResolver(function ($root, array $args): array {
```

Comment:
You can examine the result of calling `buildSchema()` directly instead of having to indirectly test it via queries. This feature is strictly schema manipulation, no need to do any actual calls for testing it.